bundles/org.openhab.core.io.rest.auth/src/main/java/org/openhab/core/io/rest/auth/internal/AuthFilter.java

```diff
@@ -75,6 +80,19 @@
     private boolean allowBasicAuth = false;
     private boolean implicitUserRole = true;
 
+    private static HashMap<ByteBuffer, UserSecurityContext> authCache = new HashMap<ByteBuffer, UserSecurityContext>();
+    private static final String UUID_STRING = UUID.randomUUID().toString();
```

Comment:
one could remove the '-' from the UUID because they don't add anything to the entropy